playbooks/tasks/portal-role-task.yml

```diff
@@ -365,3 +365,10 @@
       command: crontab -u {{ webportal_user }} {{ webportal_cron_file }}
 
   when: webportal_setup_health_checks
+
+# Pull cypress image to avoid lazily pulling it on the fly when the abuse
+# scanner needs it
+- name: Pull cypress docker image
```

Comment:
Why is this here and not part of the abuse scanner?

Reply:
`cypress` image is ran from the container on the host itself, so it is preferred to have the cypress image be available on all servers instead of pulling it in on the fly